asn1/Stream.asn

```diff
@@ -174,4 +175,12 @@ StreamDataBlocked ::= SEQUENCE {
   maxOffset VarUInt
 }
 
+StreamReceipt ::= SEQUENCE {
+  -- Identifier of the stream
+  streamId VarUInt,
+
+  -- Receipt for verifying the total amount received on this stream
+  receipt VarBytes
```

Comment:
Is there a formal ASN.1 definition for `receipt`? I see it documented in the markdown for IL-RFC-29, but don't see anything formal in this file.

Reply:
Resolving this - it's there now in Stream.asn.